Taarafo.Core.Tests.Acceptance/Apis/Comments/CommentsApiTests.Logic.cs

```diff
@@ -30,5 +32,25 @@ public async Task ShouldPostCommentAsync()
             actualComment.Should().BeEquivalentTo(expectedComment);
             await DeleteCommentAsync(actualComment);
         }
+
+        [Fact]
+        public async Task ShouldGetAllCommentsAsync()
+        {
+            // given
+            List<Comment> randomComments = await CreateRandomCommentsAsync();
+
```

Comment:
You don't need the extra empty line if they are both one-liners

Reply:
@hassanhabib fixed